CONTRIBUTING.md

```diff
@@ -83,13 +83,36 @@ Please follow the below conventions when contributing to this project.
 * Each .NET project should have its corresponding test project
 * Each class should have its corresponding test class
 * Each internal and public function in the class should be tested in the unit test class
-* Follow the below naming conventions
+* Follow the below naming conventions:
   * Test Project: *{project name}.(Unit/Functional)Tests* selecting the appropriate type of tests found in the project
   * Test Class: *{class name}Tests.cs*
   * Test methods:
     * (Data)TestMethod: `{method name}_{what is being tested}_{expected outcome}`
     * DataRow (display name): Short description that clearly differentiate between the DataRows
 
+## Code Review Process
+
+### Before Creating a Pull Request
+* The changes cannot break any existing functional/unit tests that are part of the central repository.
+  * This includes all tests, even those not associated with the given feature area.
+* Code submitted must have [basic unit test coverage](https://github.com/Azure/template-analyzer/blob/development/CONTRIBUTING.md#tests), and have all the unit tests pass. Functional tests are encouraged.
```

Comment:
That section of the Contributing doc doesn't mention built-in rule tests: https://github.com/Azure/template-analyzer/blob/eaf5661dab6d2d5b9c52098187c8d40fdb9e3013/CONTRIBUTING.md#contributing-analyzer-rules Do we want to link to that somewhere too?